<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test_detect_pair_screen</class>
 <widget class="QWidget" name="test_detect_pair_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#E5E5E5</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="8,2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="containerScreen" native="true">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="screen2">
        <property name="styleSheet">
         <string notr="true">background-color: #AFF;font-weight:bold;</string>
        </property>
        <property name="text">
         <string>SCREEN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="screen1">
        <property name="styleSheet">
         <string notr="true">background-color: #AFF;font-weight:bold;</string>
        </property>
        <property name="text">
         <string>SCREEN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="screen4">
        <property name="styleSheet">
         <string notr="true">background-color: #AFF;font-weight:bold;</string>
        </property>
        <property name="text">
         <string>SCREEN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
         <item alignment="Qt::AlignTop">
          <widget class="QGroupBox" name="groupCbDisplayType">
           <property name="title">
            <string>Display type:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignTop">
             <widget class="QComboBox" name="cbDisplayType">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">height:22px</string>
              </property>
              <item>
               <property name="text">
                <string>Item 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Item 2</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="containerConfig" native="true">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #EEEEEE</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,9">
      <item>
       <widget class="QLabel" name="lblTitle">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">text-align:center;
font-weight:bold;</string>
        </property>
        <property name="text">
         <string>TEST AND DEFINE SAMPLE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="containerParam" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,6,2">
         <property name="spacing">
          <number>7</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="containerLeft" native="true">
           <property name="styleSheet">
            <string notr="true">#containerLeft {
	border: 1px solid #A5A5A5
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10"/>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="containerMid" native="true">
           <property name="styleSheet">
            <string notr="true">#containerMid {
	border: 1px solid #A5A5A5
}</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2"/>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="containerRight" native="true">
           <property name="styleSheet">
            <string notr="true">#containerRight {
	border: 1px solid #A5A5A5
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="btnCapture">
              <property name="text">
               <string>CAPTURE</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="containerVerticalBtn" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_17">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnSaveSample">
                 <property name="text">
                  <string>SAVE SAMPLE</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignTop">
                <widget class="QPushButton" name="btnRetakeSample">
                 <property name="text">
                  <string>RETAKE SAMPLE</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="containerNavBtn" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnBack">
                 <property name="text">
                  <string>BACK</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnNext">
                 <property name="text">
                  <string>NEXT</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
